tutorials/preprocessing/plot_70_fnirs_processing.py

```diff
@@ -0,0 +1,199 @@
+# View consistency of responses across channels
+# ---------------------------------------------
+#
+# Simillarly we can view how consistent the response was across the optode
+# pairs that we selected. All the channels in this data we located over the
+# motor cortex, and all channels show a similar pattern in the data.
+
+fig, (cont, tapping) = plt.subplots(nrows=2, ncols=2, figsize=(15, 6))
+clims = dict(hbo=[-20, 20], hbr=[-20, 20])
+epochs['Control'].average().plot_image(axes=[cont[0], tapping[0]],
+                                       show=False, clim=clims)
+epochs['Tapping'].average().plot_image(axes=[cont[1], tapping[1]],
+                                       show=False, clim=clims)
+fig.show()
+
+
+###############################################################################
+# Plot standard fNIRS response image
+# ----------------------------------
+#
+# Finally we generate the most common visualisation of fNIRS data, plotting
+# both the HbO and HbR on the same figure to illustrate the relation between
+# the two signals.
+
+evoked_dict = {'Tapping/HbO': epochs['Tapping'].average(picks='hbo'),
+               'Tapping/HbR': epochs['Tapping'].average(picks='hbr'),
+               'Control/HbO': epochs['Control'].average(picks='hbo'),
+               'Control/HbR': epochs['Control'].average(picks='hbr')
+               }
+
+# Rename channels until the encoding of frequency in channel name
+# issue is fixed
+picks = _picks_to_idx(evoked_dict['Tapping/HbO'].info, 'fnirs')
+for ii in picks:
+    evoked_dict['Tapping/HbO'].rename_channels({
+        evoked_dict['Tapping/HbO'].info['chs'][ii]['ch_name']:
+            evoked_dict['Tapping/HbO'].info['chs'][ii]['ch_name'][:-4]})
+picks = _picks_to_idx(evoked_dict['Tapping/HbR'].info, 'fnirs')
+for ii in picks:
+    evoked_dict['Tapping/HbR'].rename_channels({
+        evoked_dict['Tapping/HbR'].info['chs'][ii]['ch_name']:
+            evoked_dict['Tapping/HbR'].info['chs'][ii]['ch_name'][:-4]})
+picks = _picks_to_idx(evoked_dict['Control/HbO'].info, 'fnirs')
+for ii in picks:
+    evoked_dict['Control/HbO'].rename_channels({
+        evoked_dict['Control/HbO'].info['chs'][ii]['ch_name']:
+            evoked_dict['Control/HbO'].info['chs'][ii]['ch_name'][:-4]})
+picks = _picks_to_idx(evoked_dict['Control/HbR'].info, 'fnirs')
+for ii in picks:
+    evoked_dict['Control/HbR'].rename_channels({
+        evoked_dict['Control/HbR'].info['chs'][ii]['ch_name']:
+            evoked_dict['Control/HbR'].info['chs'][ii]['ch_name'][:-4]})
+
+color_dict = {'Tapping/HbO': 'r',
+              'Tapping/HbR': 'b',
+              'Control/HbO': 'r',
+              'Control/HbR': 'b'}
+
+styles_dict = {"Control/HbO": {"linestyle": 'dashed'},
+               "Control/HbR": {"linestyle": 'dashed'}}
+
+mne.viz.plot_compare_evokeds(evoked_dict, combine="mean", ci=0.95,
+                             colors=color_dict, styles=styles_dict)
```

Comment:
I'm pretty sure this last code block could be simplified, but I just ran out of time. At the very least, I notice that we're using a private `_picks_to_idx` function (not ideal), and the color and styles dict should allow you to just specify the tag-based conditions, so `color_dict = dict(HbO='r', HbR='b')` and `styles_dict = dict(Control=dict(linestyle='dashed'))` ought to work.